prime/src/components/borrow/actions/AloeWithdrawActionCard.tsx

```diff
@@ -70,8 +70,14 @@ export function AloeWithdrawActionCard(prop: ActionCardProps) {
     token1.decimals
   );
 
+  const hasOutstandingBorrows = Object.values(accountState.liabilities).some((liability) => liability.isGtZero());
+
   const max = selectedTokenOption.value === TokenType.ASSET0 ? allowed0 : allowed1;
-  const maxString = max.toString(GNFormat.DECIMAL);
+  const eightyPercentMax = max.recklessMul(80).recklessDiv(100);
+  const maxString = hasOutstandingBorrows
+    ? eightyPercentMax.toString(GNFormat.DECIMAL)
+    : max.toString(GNFormat.DECIMAL);
```

Comment:
```suggestion
  const maxString = (hasOutstandingBorrows ? eightyPercentMax : max).toString(GNFormat.DECIMAL);
```

Reply:
While your suggestion is more condensed, I find having it on separate lines is more readable.